docker-compose-services/meilisearch/docker-compose.meilisearch.yaml

```
@@ -0,0 +1,27 @@
+version: '0.36'
+services:
+  meilisearch:
+    container_name: ddev-${DDEV_SITENAME}-meilisearch
+    image: getmeili/meilisearch:v0.20.0
+    hostname: ${DDEV_SITENAME}-meilisearch
+    ports:
+      - 7700:7700
```

Please use `expose` instead of ports - otherwise only one of these projects can be running at a time.

```suggestion
    expose:
      - 7700
```

---

thanks for the quick review! the canfiguration has been changed to use "expose"